sway/desktop/output.c

```diff
@@ -199,6 +199,13 @@ void output_layer_for_each_toplevel_surface(struct sway_output *output,
 	wl_list_for_each(layer_surface, layer_surfaces, link) {
 		struct wlr_layer_surface_v1 *wlr_layer_surface_v1 =
 			layer_surface->layer_surface;
+
+		struct render_data *data = user_data;
+		data->deco_data.blur = layer_surface->layer != ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND ?
```

Comment:
If there's a way to check if a layer is a subsurface (off of my head I don't think there currently is), we should check here

Reply:
Might have found a way of doing this. Just noticed that the `wlr_surface_state` struct has a list of surfaces above in the tree, so something like this works :)

```c
bool is_subsurface = wl_list_length(&surface->current.subsurfaces_above) > 0;
```